packages/core/strapi/ee/index.js

```diff
@@ -2,7 +2,7 @@
 
 const { pick, isEqual } = require('lodash/fp');
 
-const { readLicense, verifyLicense, fetchLicense, LicenseCheckError } = require('./license');
+const { readLicense, verifyLicense, fetchLicense, LicenseCheckError } = require('./utils');
```

Comment:
Why did you change this filename? 🤔 Doesn't look related to the NodeJS version upgrade.

Reply:
For some reason even on posix, Node 20 thinks "../license" could be "../LICENSE" instead of "../license.js".

The alternative was to change to "../license.js" but that just seemed like asking for trouble in the future.

Reply:
OK I see, maybe we can still keep the "license" business word and put `license.utils.js`, wdyt? As utils is very generic.

Reply:
moved it to a utils folder :)